.env.example

```diff
@@ -37,7 +37,7 @@ MAIL_FROM_NAME="${APP_NAME}"
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 GOOGLE_CALENDAR_ENABLED=true
-GOOGLE_REDIRECT=http://localhost/login/google/end
+GOOGLE_REDIRECT=http://localhost:8751/login/google/end
```

Comment:
Why not `toby.blumilk.localhost`?

Reply:
Because login by Google doesn't work with `toby.blumilk.localhost`, but I can change this.

Reply:
That would be nice.